server/app/services/program/ProgramService.java

```diff
@@ -635,6 +676,13 @@ private ImmutableSet<CiviFormError> validateProgramData(
     if (displayMode.isBlank()) {
       errorsBuilder.add(CiviFormError.of(MISSING_DISPLAY_MODE_MSG));
     }
+    var validNotificationPrefereces =
+        Arrays.stream(ProgramNotificationPreference.values())
+            .map(ProgramNotificationPreference::getValue)
+            .collect(ImmutableList.toImmutableList());
+    if (notificationPreferences.stream().anyMatch(p -> !validNotificationPrefereces.contains(p))) {
```

Comment:
**typo:** prefereces

Reply:
I was gonna missspell it one of the four dozen times I wrote it in this PR 😅